doc/pgtap.mmd

```diff
@@ -122,9 +122,8 @@ schema, use the `SCHEMA` clause to specify the schema, like so:
 Testing pgTAP with pgTAP
 ------------------------
 
-In addition to the PostgreSQL-standard `installcheck` target, the `test`
-target uses the `pg_prove` Perl program to do its testing, which will be
-installed with the
+`test` target uses the `pg_prove` Perl program to do its testing, which needs
```

Comment:
I think you deleted more words here than you mean to.

Reply:
Why? From the code `test` target doesn't use `installcheck`. I thought it is a leftover.

Reply:
No, I mean you deleted the beginning of the sentence.

Reply:
Maybe there is a misunderstanding. What's the point in telling that `test` uses `installcheck` if it doesn't use it?

Reply:
You deleted the words

> In addition to the PostgreSQL-standard `installcheck` target, the 

And I don't think that's what you meant to do.

Reply:
Brought it back. Still makes the text sound wrong for me.

Reply:
LGTM.